third_party/nvfuser/csrc/transform_replay.cpp

```diff
@@ -802,7 +826,8 @@ int TransformReplay::getMatchedLeafPosWithoutReplayPasC(
 int TransformReplay::getMatchedLeafPosWithoutReplayCasP(
     const TensorView* consumer,
     const TensorView* producer,
-    int producer_pos) {
+    int producer_pos,
+    bool skip_resize) {
```

Comment:
To be more consistent, should we add a `skip_swizzle` and do the same thing in transform propagator?

Reply:
Probably yes, but I'm actually not quite sure about skipping swizzles. Looks like they are always skipped, so it's probably not necessary, but for consistency, yes. For resize, the option is necessary to have.